lib/editor-linter.coffee

```diff
@@ -77,7 +91,11 @@ class EditorLinter
 
   # This method sets or gets the lock status of given type
   _lock: (wasTriggeredOnChange, value) ->
```

Comment:
I realize this is off the topic of this PR, but weren't we trying to get rid of underscored method names?